tutorials/Python/00_Introduction/Introduction.ipynb

```diff
@@ -143,6 +143,20 @@
     "In the remainder of this tutorial we will mainly focus on executing code within the jupyter notebook. However, it is worth remembering that all this could also be written in a file and executed in a similar manner to what is shown above."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Python: An Interpreted Language"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Python is an _interpreted_ programming language. This means that Python code is read and executed line by line by the Python _interpreter_ (the program that transforms Python code you wrote into something that your computer can understand and run). This is different from _compiled_ languages (such as FORTRAN or C/C++), where all the code is first translated by the _compiler_ to something the computer can understand and then run."
```

Comment:
Do we want to add a line as to why we focus on python here (and any other similar alternatives like R, and matlab)?

Reply:
Sure, but don't blame me if then it will look like we are paid from PSF to advertise Python ;)

Reply:
Being a shill for python is an honorable pursuit :)